evalml/pipelines/components/transformers/dimensionality_reduction/lda.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as SkLDA
+
+from evalml.pipelines.components.transformers import Transformer
+from evalml.utils.gen_utils import is_all_numeric
+
+
+class LDA(Transformer):
```

Comment:
I'd vote for changing the name to `LinearDiscriminantAnalysis` because LDA could be Latent Dirichlet Allocation.